src/txn/txn_timestamp.c

```diff
@@ -65,25 +65,20 @@ __wt_timestamp_to_hex_string(
 
 /*
  * __wt_verbose_timestamp --
- *	Output a verbose message along with the specified timestamp
+ *	Output a verbose message along with the specified timestamp.
  */
 void
 __wt_verbose_timestamp(WT_SESSION_IMPL *session,
     const wt_timestamp_t *ts, const char *msg)
 {
-#ifdef HAVE_VERBOSE
 	char timestamp_buf[2 * WT_TIMESTAMP_SIZE + 1];
 
-	if (__wt_timestamp_to_hex_string(session, timestamp_buf, ts) != 0)
+	if (!WT_VERBOSE_ISSET(session, WT_VERB_TIMESTAMP) ||
```

Comment:
this check is already done by __wt_verbose() called next. We will effectively be doing this test twice now.

Reply:
The reason why I added this is reduce performance impact by avoiding the execution of "if" statement before the __wt_verbose call.

Reply:
I don't have a good solution to this issue, I think it should be okay to check twice if WT_VERB_TIMESTAMP is set.
@agorrod can you have a look at this change here, what do you think.

Reply:
I don't have a strong opinion on this. I think the code is more obvious with the explicit check, so I'm happy for it to stay the way it is here.